apis/opts/config/runtime_test.go

```diff
@@ -10,9 +10,11 @@ import (
 func TestNewRuntime(t *testing.T) {
 	assert := assert.New(t)
 
+	var tmp = map[string]types.Runtime(nil)
```

Comment:
Thank you for pointing out where I need to improve. I will pay attention to it in the future. Since there is a PR #2329 that has already add test cases, I will close my PR. Thanks for you advice anyway. Have a nice day.